test/range/count.wren

```diff
@@ -0,0 +1,25 @@
+// Ordered range.
+IO.print((2..5).count) // expect: 4
+IO.print((3..3).count) // expect: 1
+IO.print((0..3).count) // expect: 4
+IO.print((-5..3).count) // expect: 9
+IO.print((-5..-2).count) // expect: 4
+
+// Backwards range.
+IO.print((5..2).count) // expect: 4
+IO.print((3..0).count) // expect: 4
+IO.print((3..-5).count) // expect: 9
+IO.print((-2..-5).count) // expect: 4
+
+// Exclusive ordered range.
+IO.print((2...5).count) // expect: 3
+IO.print((3...3).count) // expect: 0
+IO.print((0...3).count) // expect: 3
+IO.print((-5...3).count) // expect: 8
+IO.print((-5...-2).count) // expect: 3
+
+// Exclusive backwards range.
+IO.print((5...2).count) // expect: 3
+IO.print((3...0).count) // expect: 3
+IO.print((3...-5).count) // expect: 8
+IO.print((-2...-5).count) // expect: 3
```

Comment:
@bjorn, I was already using the current implementation as a reference.

I was asking because (my fault?) I cannot really decide what should be the correct behaviour/result of the above statements.

More precisely, how can we explain those result (with the current implementation)?
```
IO.print(((3.1)..(6.1)).count) // expect: 4
IO.print(((3.1)...(6.1)).count) // expect: 3
IO.print(((0.0)..(1.9)).count) // expect: 2
IO.print(((0.0)...(1.9)).count) // expect: 2
```
Any help is appreciated.